dluhc-component-library/src/api/planningData/api.ts

```diff
@@ -1,7 +1,8 @@
 import { FeatureCollection } from "geojson";
 import WKT from "ol/format/WKT.js";
 import { Geometry } from "ol/geom";
-import { DatasetResponse } from "src/api/types";
+import { DatasetResponse } from "./types";
+import { useQuery } from "@tanstack/react-query";
```

Comment:
`Non-blocker` Can we please put library imports above absolute and relative imports. I need to write add the eslint rule for this.

Library imports
Absolute imports
Relative imports

Css import